lib/kinetic.js

```diff
@@ -558,6 +557,9 @@ class InitPDU extends PDU {
  * Getting logs and stats request following the kinetic protocol.
  * @param {number} sequence - monotonically increasing number for each
  *                                request in a TCP connection.
+ * @param {number} connectionID - the connection ID received from the InitPDU
+ * @param {number} clusterVersion - the version of the cluster received from the
+                                    InitPDU
  * @param {Object} options - optional :
```

Comment:
It is used if you want to know what type of log you want. But you are right it can be remove to just have an array instead of an object with just an array inside